faq/pathogen-subs-guide.rst

```diff
@@ -263,58 +263,60 @@ for permitted values.
 for permitted values.
 
+Note that ‘chromosome’ should here be understood as a general term for a range of complete replicons, including chromosomes of eukaryotes, prokaryotes, 
+and viruses, as well as organellar chromosomes and plasmids. All of these may be submitted within the same chromosome-level assembly.
+
+If your assembly is not complete, you can submit a **contig** or **scaffold** level assembly. The `Assembly levels <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_ tutorial page explains the assembly levels recognised by ENA.
+
 **chromosome_list.txt** examples
 
-Viruses
---------
+.. tabs::
+
+   .. tab:: Viruses
+
+      The chromosome topology for viral sequences can be specified in the manifest file as **MOLECULETYPE** : ‘genomic DNA’, ‘genomic RNA’ or ‘viral cRNA’.
+      By default the chromosome topology will be assumed to be linear, therefore if the topology is cicrular, it must be specified.
 
-.. code-block:: xml
+      .. code:: none
 
-   chr01   1 Monopartite
+         chr01   1 Monopartite
 
-.. code-block:: bash
+      .. code:: none
 
-   chr01   1 Linear-Monopartite viroid
+         chr01   1 Linear-Monopartite viroid
 
-.. code-block:: python
+      .. code:: none
 
-   chr01   1 Multipartite
-   chr02   2 Multipartite
+         chr01   1 Multipartite
+         chr02   2 Multipartite
 
-Bacteria
----------
+   .. tab:: Bacteria
 
-By default prokaryotic chromosomes and plasmids will be assumed to reside in the in the cytoplasm, however, the 'plasmid'
-chromosome_location may be specified.
-By default the chromosome topology will be assumed to be linear, so in this example the circular topology was specified.
+      By default prokaryotic chromosomes and plasmids will be assumed to reside in the in the cytoplasm, however, the 'plasmid'
+      **CHROMOSOME_LOCATION** may be specified.
+      By default the **TOPOLOGY** will be assumed to be linear, so in this example the circular topology was specified.
 
-.. code-block:: xml
+      .. code:: none
 
-   chr01   1 circular-Chromosome
-   chr02   2 circular-plasmid
-   chr03   3 circular-plasmid
+         chr01   1 circular-Chromosome
+         chr02   2 circular-Chromosome plasmid
+         chr03   3 circular-Chromosome plasmid
 
-Eukaryota
----------
+   .. tab:: Eukaryota
 
-By default eukaryotic chromosomes will be assumed to reside in the nucleus. By default the chromosome topology
-will be assumed to be linear, but it may also be specified.
+      By default eukaryotic chromosomes will be assumed to reside in the nucleus. By default the chromosome topology
+      will be assumed to be linear, but it may also be specified.
```

Comment:
Why was it **`TOPOLOGY`** in the bacteria tab, but just `topology` here?

Reply:
this has been corrected to be consistent